quarkus-debezium-parent/quarkus-debezium-engine-parent/runtime/src/test/java/io/quarkus/debezium/configuration/DebeziumConfigurationEngineParserTest.java

```diff
@@ -68,6 +69,11 @@ public Map<String, String> configurations() {
                             }
                         });
             }
+
+            @Override
+            public Map<String, DevServicesConfig> devservices() {
+                return Map.of();
+            }
```

Comment:
What would you say to making the methods on the `Capturing` interface have defaults to aviod the need to define these _empty_ methods implementations if they're not needed?